src/fuzz/meson.build

```diff
@@ -116,4 +116,17 @@ fuzzers += [
         [['src/fuzz/fuzz-bus-label.c'],
          [libshared],
          []],
+
+        [['src/fuzz/fuzz-env-file.c'],
+         [libshared],
+         []],
+
+        [['src/fuzz/fuzz-hostname-util.c'],
+         [libshared],
+         []],
+
+        [['src/fuzz/fuzz-nspawn-settings.c'],
+         [libshared,
+          libnspawn_core],
+         []],
```

Comment:
I'd prefer to have this split among the three commits that add those files, like it is done with the first commit.